### ISSUE TITLE
Problem with libclang.dll on Windows 11

### ISSUE THREAD

**Issue by mcprotector** [zmwangx/rust-ffmpeg#169]
Windows 11
Rust for Visual Studio

That message in build info:
Unable to find libclang: "couldn't find any valid shared libraries matching: ['clang.dll', 'libclang.dll'], set the `LIBCLANG_PATH` environment variable to a path where one of these files can be found (invalid: [])

A lot of versions for clang downloaded, LIBCLANG_PATH set, but don't work.

**Comment by EpicCodeWizard** [comment]
@mcprotector

**Comment by mcprotector** [comment]
Yes, I managed to solve this problem. It turned out that we needed to enter the path directly into the System.

Now I have another problem. Immediately after ffmpeg is initiated, system messages cannot be displayed on the console screen. And I basically don’t see what’s going on inside there.

**Comment by EpicCodeWizard** [comment]
Not sure what you mean, can you provide some error logs?

**Comment by mcprotector** [comment]
> Not sure what you mean, can you provide some error logs?

There is no errors.
Simply `ffmpeg::init().unwrap();` block `println` output.
Without I can echo messages to console. When ffmpeg is inited console have no messages.

`println!("Hello World!");`
output "Hello World!"

```
ffmpeg::init().unwrap();
println!("Hello World!");
```
empty output.
